src/visip/dev/type_inspector.py

```diff
@@ -1,22 +1,23 @@
 import typing
 import typing_inspect
 import enum
-from .dtype import valid_base_types, Constant, DataClassBase
+from .dtype import DataClassBase
```

Comment:
Is TypeInspector still in use? Should use inspection capabilities of the dtypes.